lib/elixir/lib/macro.ex

```diff
@@ -585,6 +585,9 @@ defmodule Macro do
   end
 
   # Access
+  def to_string({{:., _, [Access, :get]}, _, [{_, _, [_|_]} = left, right]} = ast, fun) do
+    fun.(ast, "(" <> to_string(left, fun) <> ")" <> to_string([right], fun))
+  end
```

Comment:
The newline is needed to match the style with the rest of `to_string` definitions.